databricks/koalas/namespace.py

```diff
@@ -538,6 +540,216 @@ def column_name(value):
     return kdf[remaining_columns]
 
 
+# TODO: there are many parameters to implement and support. See Pandas's pd.caoncat.
+def concat(objs, axis=0, join='outer', ignore_index=False):
```

Comment:
We don't need to add `axis`?

Reply:
Current behaviour is when axis is `0`. Let me throw an exception if the values are different.